yamale/validators/validators.py

```diff
@@ -21,7 +12,236 @@ class String(Validator):
     constraints = [con.LengthMin, con.LengthMax, con.CharacterExclude]
 
     def _is_valid(self, value):
-        return isstr(value)
+        return util.isstr(value)
+
+
+class Pathname(String):
```

Comment:
> Inheriting also means you bring `String`'s constraints along which doesn't really apply to `Pathname`.

That's _totally a valid concern. I'll reinherit `Pathname` directly from `Validator` then. Thanks for the thoughtful heads up!